app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -222,6 +226,7 @@ public void onCreate(Bundle savedInstanceState) {
         emojiButton = findViewById(R.id.composeEmojiButton);
         hideMediaToggle = findViewById(R.id.composeHideMediaButton);
         emojiView = findViewById(R.id.emojiView);
+        emojiList = new ArrayList<>();
```

Comment:
1. You could do `Collections.emptyList()`. Not that important but good-to-know
2. We have redundant checks for it to be null in the fragment

Reply:
The json converter requires it to be nonnull, however - I hit this while working on tests

Reply:
Yes, I I'm cool with making it non-null at all times, I'm just saying that we have checks for null which we don't need now